checker/build.xml

```diff
@@ -550,6 +550,8 @@
               </fileset>
           </batchtest>
       </junit>
+
+      <antcall target="units_extension-tests"/>
```

Comment:
I'm guessing the underscores were an attempt to indicate the parsing.
I don't have a problem with underscores.  But if you are going to remove some, then yes, please remove them all.  Thanks!